frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -91,23 +95,11 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
     <>
       <ApplicationsPage
         breadcrumb={
```

Comment:
why did you need to create a bunch of new components here?

Reply:
Yeah, this is unnecessary now. Initially, I created a new ApplicationsPage component for unsupported pipelines and was reusing this breadcrumb component. However, I rolled back to use the existing ApplicationsPage. This new component is not required anymore. Do you suggest I undo this change or keep it as is because the breadcrumb component is little bigger and having separate file is fine.

Reply:
i would undo it